pennylane/drawer/tape_text.py

```diff
@@ -61,11 +61,13 @@ def _add_op(op, layer_str, wire_map, decimals):
 }
 
 
-def _add_measurement(m, layer_str, wire_map, decimals):
+def _add_measurement(m, layer_str, wire_map, decimals, cache):
     """Updates ``layer_str`` with the ``m`` measurement."""
     layer_str = _add_grouping_symbols(m, layer_str, wire_map)
 
-    obs_label = None if m.obs is None else m.obs.label(decimals=decimals).replace("\n", "")
+    obs_label = (
+        None if m.obs is None else m.obs.label(decimals=decimals, cache=cache).replace("\n", "")
+    )
```

Comment:
I think this was just a matter of the line getting too long and black formatting it this way. Since it has become multiple lines, I will rewrite it to not use single-line syntax.

Reply:
Ahh okay, I see now that its not explicitly being casted as a tuple, so it should be fine if you want to leave it as is.